npm/cmd/start.go

```diff
@@ -37,11 +38,12 @@ func newStartNPMCmd() *cobra.Command {
 			viper.AutomaticEnv() // read in environment variables that match
 			viper.SetDefault(npmconfig.ConfigEnvPath, npmconfig.GetConfigPath())
 			cfgFile := viper.GetString(npmconfig.ConfigEnvPath)
+
 			viper.SetConfigFile(cfgFile)
 
 			// If a config file is found, read it in.
 			if err := viper.ReadInConfig(); err == nil {
```

Comment:
Add a comment here saying there is no config merging, and change in bhevaior needs an explicit true for backward compability.